modules/local/get_synteny_genomes.nf

```diff
@@ -1,30 +1,35 @@
-    process GET_SYNTENY_GENOMES {
-        tag "${assembly_classT}"
-        label "process_low"
+process GET_SYNTENY_GENOMES {
```

Comment:
This feels similar to your GrabFiles, @priyanka-surana . Do you think this process should be embedded into the relevant subworkflow like you need ? Or at least get `executor = local` added here or through `modules.config`

Reply:
The important distinction is that in [`GrabFiles`](https://github.com/sanger-tol/genomenote/blob/dev/subworkflows/local/genome_statistics.nf) nothing is actually run, the main function happens in the `output` statement. The script that is submitted to the `local` executor is 
```
#!/bin/bash -euo pipefail
true
```

Here, an actual process `cp` is run which requires resources and depending on the size of the `fasta` file, so I think we should actually make this a local module. 

When I submit a command/process, I think that justifies a module. Let me know what you think and we can refine the rule going forward.

Reply:
My feeling is that the `cp` itself is not mandatory and that `ln` would work equally well. I don't expect the downstream processes to require their input files to be copies rather than the original files. Or if that's the case, that needs to be very carefully documented !

That's why I see the module as just a convenient way of selecting specific files from a directory into a channel, which is exactly what GrabFiles does.

Reply:
In that case, yes they can do exactly like GrabFiles, they do not even need a `ln`. The treeval team can decide how they would like to proceed.

Reply:
Discussed with Damon and Yumi and we decided to keep get_synteny_genomes seperate module.